src/utils.ts

```diff
@@ -0,0 +1,67 @@
+import * as core from "@actions/core";
+import { read } from "@1password/op-js";
+import {
+	authErr,
+	envConnectHost,
+	envConnectToken,
+	envServiceAccountToken,
+	envManagedVariables,
+} from "./constants";
+
+export const semverToInt = (input: string): string =>
```

Comment:
Looks familiar 😛

Reply:
Opened this PR to make it available directly from `op-js` (and thus no longer duplicating the code here):
 [Export semverToInt](https://github.com/1Password/op-js/pull/126).

Reply:
🙌

Reply:
Your PR has been merged and a new version of op-js has been [published](https://www.npmjs.com/package/@1password/op-js). Update to 0.1.9 to use the new semverToInt utility :)

Reply:
@edif2008 Unresolving this thread because it we still need to:

1. Upgrade to the latest version of `op-js`.
2. Remove the `semverToInt` function + unit tests and replace it with the imported one from `op-js`.

Reply:
Solved with [57ddba0](https://github.com/1Password/load-secrets-action/pull/36/commits/57ddba0a994c9e199fa23491b9239fe3005e695a) and [35650f8](https://github.com/1Password/load-secrets-action/pull/36/commits/35650f8988cd23393a4e01663ccc6cd677368432).